vmm/sandbox/src/client.rs

```diff
@@ -37,6 +41,8 @@ use vmm_common::api::{sandbox::*, sandbox_ttrpc::SandboxServiceClient};
 use crate::network::{NetworkInterface, Route};
 
 const HVSOCK_RETRY_TIMEOUT: u64 = 10;
+const TIME_SYNC_PERIOD: u64 = 60;
```

Comment:
TIME_SYNC_PERIOD_IN_SEC

Reply:
We assume that default unit is second if not specifiy. Only variables that use other unit shoule append suffix.